xml/containers-basics.xml

```diff
@@ -66,16 +63,19 @@
   </listitem>
   <listitem>
    <para>
-     A given container will encapsulate binaries for a given architecture (x86_64 or aarch64 for 
-     instance). So a container made for x86_64 will only run on a x86_64 system host.
+     A container encapsulates binaries for a specific architecture (x86_64 or
+     aarch64 for instance). So a container made for x86_64 only runs on a
+     x86_64 system host.
    </para>
   </listitem>
   <listitem>
    <para>
-    The security capabilities of containers are often overestimated and the overall security depends
-    on the host system. Containerized applications can be secured through &aa; or &selnx;
-    profiles. Container security would require implementing tools and policies to ensure that the
-    container infrastructure and applications are secure.
+    Containers in itself are no more secure than executing binaries outside of a
```

Comment:
This paragraph contains the word "container" in one form or another 4 times, I would suggest to rephrase this a bit as it reads a bit clunky.

Reply:
Ugh, good point!